### ISSUE TITLE
transform.Normalize has wrong inputs in Part 2 - Neural Networks in PyTorch (Exercises).ipynb

### ISSUE THREAD

**Issue by rawwar** [udacity/deep-learning-v2-pytorch#10]
Hi, 
In  Part 2 - Neural Networks in PyTorch (Exercises).ipynb, inside the second code cell, the following code works fine, but, logically wrong
```
transform = transforms.Compose([transforms.ToTensor(),
                              transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5)),
                              ])
```

it should be 
```
transform = transforms.Compose([transforms.ToTensor(),
                              transforms.Normalize((0.5,), (0.5, )),
                              ])
```

reason being that MNIST has only one color channel and what you mentioned is for an image with 3 channels

**Comment by mcleonard** [comment]
You're right, that is cleaner. Fixed here: 5fd18e2f4d12434420514b4011b52446d45b6f77

Thanks!
